monitoring/tests/system/test_vpcsc.py

```diff
@@ -0,0 +1,244 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright 2019 Google LLC
+# 
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+# 
+#     https://www.apache.org/licenses/LICENSE-2.0
+# 
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+# DO NOT MODIFY! AUTO-GENERATED!
+# This file is auto-generated on 2019-04-23.
+
+
+import os
+
+from google.api_core import exceptions
+from google.cloud import monitoring_v3
+from google.cloud.monitoring_v3 import enums
+
+
+class TestVPCServiceControlV3(object):
+
+    @staticmethod
+    def _is_rejected(call):
+        try:
+            responses = call()
+
+            # If we reach this line, then call() did not raise. The return
+            # result must be either a google.api_core.page_iterator.Iterator
+            # instance, or None.
+            list(responses)
+        except exceptions.PermissionDenied as e:
+            return e.message == "Request is prohibited by organization's policy"
+        except:
+            pass
+        return False
+
+    @staticmethod
+    def _do_test(delayed_inside, delayed_outside):
+        if ('GOOGLE_CLOUD_TESTS_IN_VPCSC' in os.environ) and (os.environ['GOOGLE_CLOUD_TESTS_IN_VPCSC'].lower() == 'True'.lower()):
```

Comment:
You can just write `os.environ.get('GOOGLE_CLOUD_TESTS_IN_VPCSC', 'false').lower() == 'true'`.